chroma-manager/tests/integration/shared_storage_configuration/test_connectivity.py

```diff
@@ -10,7 +10,7 @@
 MAX_AGENT_BACKOFF = 60
 
 
-@skipIf(not config.get('simulator'), "RealRemoteOperations can't fake out network failures")
+@skip("Needs setup on real hardware")
```

Comment:
I won't ditto all of these, but the question about tickets can be assumed for all of them.

Reply:
https://github.com/intel-hpdd/intel-manager-for-lustre/issues/520